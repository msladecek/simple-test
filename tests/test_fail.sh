#!/usr/bin/env sh
COLUMNS=16 ./simple-test examples/fail/*.py
