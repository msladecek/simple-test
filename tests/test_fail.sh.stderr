examples/fail/test_exit_code.py checking exit code

Expected:
100
Actual:
2
----------------
examples/fail/test_hello_world.py checking stdout

Expected:
hello world

Actual:
ahoy matey

----------------
examples/fail/test_no_shebang.py not executable
examples/fail/test_output_streams.py checking stdout

Expected:
hello on stdout!

Actual:
hello on stdout!
bonjour sur stderr!

----------------
examples/fail/test_output_streams.py checking stderr

Expected:
bonjour sur stderr!

Actual:

----------------
